themes/bootstrap3/js/common.js

```diff
@@ -619,22 +619,12 @@ function setupQRCodeLinks(_container) {
   var container = _container || $('body');
 
   container.find('a.qrcodeLink').click(function qrcodeToggle() {
-    if ($(this).hasClass("active")) {
-      $(".result-link-label", this).html(VuFind.translate('qrcode_show'));
-      $(this).removeClass("active");
-    } else {
-      $(".result-link-label", this).html(VuFind.translate('qrcode_hide'));
```

Comment:
The reason this wasn't working is that the qrcode translation strings were not being passed through in layout/js-translations.phtml. Would it be better to fix that rather than to remove the hide translation? (I don't have strong feelings one way or another, but just raising the question).

Reply:
I decided to remove it for consistency with all other dropdowns that don't change the text. But it could be added back if that's better.

Reply:
Makes sense to me, but I'm going to ask @crhallberg for an additional review in case he has any further thoughts or opinions. :-)

Reply:
Thanks, @crhallberg. If @EreMaijala doesn't mind putting back the text switching, it would greatly reduce the number of files touched by this PR and reduce the behavior changes as well. But if he feels strongly that it's better this way, he already has approvals from both of us and is free to hit the merge button. I'll leave the final decision up to him! :-)

Reply:
The problem with text switching, as I see it, is that it makes the aria attributes a bit weird. If you have `aria-expanded="true"` on a toggle that says "Hide QR Code", it just doesn't make a whole lot of sense. Maybe it would be best to remove the verb and just display e.g. "QR Code ▼"? We already have a translation for "QR Code", so no new language strings would be needed.

Reply:
@EreMaijala, that sounds like a great compromise to me, and I think it probably lets us get rid of a second translation string. :-)

Reply:
Done. :)